flink-runtime/src/main/java/org/apache/flink/runtime/util/BashJavaUtils.java

```diff
@@ -74,13 +70,8 @@ private static Configuration getConfigurationForStandaloneTaskManagers(String[]
 	 */
 	public enum Command {
 		/**
-		 * Get dynamic configs of task executor resources.
-		 */
-		GET_TM_RESOURCE_DYNAMIC_CONFIGS,
-
-		/**
-		 * Get JVM parameters of task executor resources.
+		 * Get JVM parameters and dynamic configs of task executor resources.
 		 */
-		GET_TM_RESOURCE_JVM_PARAMS
+		GET_TM_RESOURCE_PARAMS
```

Comment:
we could just remove the concept of commands.

Reply:
I prefer to keep it in case we add other utility functions in this class.

Reply:
If this were to happen we'd revert back to having to call it multiple times; we'd likely just print another line instead, no?

Reply:
I think we merge the two command here because they are all used in the same place when tm starts. However, the scope of this class is utility for bash. Commands added in future might be called anywhere. In that case, I think it's not just appending another line.